Gruntfile.js

```diff
@@ -36,6 +36,8 @@ const JS_FILES = [
 	`${SRC_DIR}/connectors/**/*.js`,
 	// Core files
 	`${SRC_DIR}/core/**/*.js`, `${SRC_DIR}/ui/**/*.js`,
+	// Shell Scripts
+	`${SRC_DIR}/scripts/*.js`,
```

Comment:
You will have this path as `src/scripts/*.js` after it's expanded. You should use `scripts/*.js` instead.

Reply:
I believe it is fixed.